src/test/java/com/puppycrawl/tools/checkstyle/ConfigurationLoaderTest.java

```diff
@@ -695,4 +696,27 @@ public void testConfigWithIgnoreExceptionalAttributes() {
         }
     }
 
+    @Test
+    public void testLoadConfiguration3() throws Exception {
+        final String[] configFiles = {
+            "InputConfigurationLoaderOldConfig0.xml",
+            "InputConfigurationLoaderOldConfig1.xml",
+            "InputConfigurationLoaderOldConfig2.xml",
+            "InputConfigurationLoaderOldConfig3.xml",
+            "InputConfigurationLoaderOldConfig4.xml",
+            "InputConfigurationLoaderOldConfig5.xml",
+            "InputConfigurationLoaderOldConfig6.xml",
+            "InputConfigurationLoaderOldConfig7.xml",
+        };
+
+        for (String configFile : configFiles) {
+            final DefaultConfiguration config =
+                    (DefaultConfiguration) ConfigurationLoader.loadConfiguration(
+                            new InputSource(Files.newInputStream(Paths.get(
+                                    getPath(configFile)))),
+                            new PropertiesExpander(new Properties()),
+                            IgnoredModulesOptions.OMIT);
+            assertThat(config.getPropertyNames()).asList().isEmpty();
```

Comment:
I do not understand this assert.
please use getChildren() , config  should have not 0 amount of modules that are loaded.